### ISSUE TITLE
Master Password Re-Prompt Validation Broken?

### ISSUE THREAD

**Issue by Smashing9246** [bitwarden/clients#6268]
### Steps To Reproduce

1. Navigate to an item in vault for which Master Password Re-Prompt is enabled.
2. Take an action that brings up the Master Password Re-Prompt window.
3. Type any input (correct or incorrect) into the provided field and push the "Enter/Return" key or click "Ok".

### Expected Result

- Be denied access to the item for an incorrect input
- Be granted access to the item with a correct input

### Actual Result

- PC Desktop and Chrome Browser extension: zero response or feedback from window; cannot proceed
- Android mobile app: crashes

### Additional Context

As I haven't used Github to any great extent, I am unsure how to include multiple labels for this issue.

Chrome is Version 114.0.5735.199 (Official Build) (32-bit)
Build is Version 2023.8.3

The mobile operating system is Android version 13
The mobile app version I have is 2023.8.0 (7466)

### Operating System

Windows

### Operating System Version

Version	10.0.19045 Build 19045

### Installation method

Direct Download (from bitwarden.com)

### Build Version

Version 2023.8.3

### Issue Tracking Info

- [X] I understand that work is tracked outside of Github. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by gleichd** [comment]
I believe I've encountered the same bug.

Trying to unlock the vault silently fails.

I'm running FreeBSD(v13.2-RELEASE-p3)
Bitwarden plugin (v2023.8.3)

This is a log from Chromium (v116.0.5845.140)
```
TypeError: Failed to execute 'digest' on 'SubtleCrypto': The provided value is not of type '(ArrayBuffer or ArrayBufferView)'.
    at xn.<anonymous> (chrome-extension://nngceckbapebfimnlniiiahkandclblb/background.js:1:150070)
    at Generator.next (<anonymous>)
    at chrome-extension://nngceckbapebfimnlniiiahkandclblb/background.js:1:148086
    at new Promise (<anonymous>)
    at Dn (chrome-extension://nngceckbapebfimnlniiiahkandclblb/background.js:1:147831)
    at xn.hash (chrome-extension://nngceckbapebfimnlniiiahkandclblb/background.js:1:149833)
    at Vc.<anonymous> (chrome-extension://nngceckbapebfimnlniiiahkandclblb/background.js:1:584693)
    at Generator.next (<anonymous>)
    at chrome-extension://nngceckbapebfimnlniiiahkandclblb/background.js:1:568055
    at new Promise (<anonymous>)", source: chrome-extension://nngceckbapebfimnlniiiahkandclblb/popup/vendor-angular.js (2)
```

**Comment by panchr** [comment]
This also broken for me on mobile. The app crashes after entering the prompt.

**Comment by F0RG0TU53RN4M3** [comment]
Encountering same issue on MacOS using Safari on any website, as well as when using the MacOS desktop app. However, when using the browser extension on Safari on the actual Bitwarden site, it works??

**Comment by Yu-Qingling** [comment]
Logging out  and then logging in again, can use it for a while, android and extension. But web pages and Win11 applications are normal.

**Comment by Yu-Qingling** [comment]
Logging out  and then logging in again, can use it for a while, android and extension. But web pages and Win11 applications are normal.

**Comment by jolo-dev** [comment]
I am experiencing the same problem. When attempting to use the desktop application, nothing happens. On the Android mobile app, the app crashes when re-entering the master password for that item.

**Comment by djsmith85** [comment]
Duplicate of https://github.com/bitwarden/clients/issues/6153

**Comment by djsmith85** [comment]
Duplicate of https://github.com/bitwarden/mobile/issues/2733

**Comment by djsmith85** [comment]
Hi @Smashing9246, @gleichd, @panchr, @F0RG0TU53RN4M3, @Yu-Qingling and @jolo-dev 

Thank you for your reports. This has previously been reporting with https://github.com/bitwarden/clients/issues/6153 for web/desktop/browser with a fix which will be included in the upcoming `2023.9`-release

For mobile this has previously been reported with https://github.com/bitwarden/mobile/issues/2733 which has also received a fix to be included in the `2023.9` release.

Closing this now.
